plugin/lsp.vim

```diff
@@ -1,4 +1,4 @@
-if exists('g:lsp_loaded')
+if exists('g:lsp_loaded') || !exists('*json_encode')
```

Comment:
Should we also check for `timers` and `lambda` support?

Reply:
ok